salt/modules/boto3_route53.py

```diff
@@ -719,24 +720,20 @@ def delete_hosted_zone_by_domain(Name, PrivateZone=None, region=None, key=None,
     Id = zone[0]['HostedZone']['Id']
     return delete_hosted_zone(Id=Id, region=region, key=key, keyid=keyid, profile=profile)
 
+
 def aws_encode(x):
     '''
     An implementation of the encoding required to suport AWS's domain name
-    rules defined here:
-
-        .. __: http://docs.aws.amazon.com/Route53/latest/DeveloperGuide/DomainNameFormat.html
+    rules defined `here <http://docs.aws.amazon.com/Route53/latest/DeveloperGuide/DomainNameFormat.html>`_.
```

Comment:
Sphinx will not permit links defined using the format `` `link text <url>`_`` when there are more than one link in the same file with the same link text but different target URLs. Instead, you can do:

```
rules defined here__:

.. __: http://docs.aws.amazon.com/Route53/latest/DeveloperGuide/DomainNameFormat.html
```

Reply:
Sure thing, sorry for all the iterations here